charts/metallb/README.md

```diff
@@ -1,6 +1,6 @@
 # metallb
 
-![Version: 0.9.6](https://img.shields.io/badge/Version-0.9.6-informational?style=flat-square) ![Type: application](https://img.shields.io/badge/Type-application-informational?style=flat-square) ![AppVersion: v0.9.6](https://img.shields.io/badge/AppVersion-v0.9.6-informational?style=flat-square)
+![Version: 0.12.2](https://img.shields.io/badge/Version-0.12.2-informational?style=flat-square) ![Type: application](https://img.shields.io/badge/Type-application-informational?style=flat-square) ![AppVersion: v0.12.2](https://img.shields.io/badge/AppVersion-v0.12.2-informational?style=flat-square)
```

Comment:
Using this comment for the whole file.  It looks like there's a bunch of fields being added to README.md that's not directly related to the PR.  It might be better to put those in a separate PR.

Reply:
There are number of params that weren't documented. Helm-docs updated them all. Happy to create a separate MR to update only the README, but the readme right now is out of step with the chart.

Reply:
changed the version here to 0.0.0 to match with the chart version. Or do yo prefer I leave it at 0.9.6?